src/Compilers/CSharp/Portable/Compilation/InitializerSemanticModel.cs

```diff
@@ -28,6 +28,7 @@ internal sealed class InitializerSemanticModel : MemberSemanticModel
                                      int speculatedPosition = 0) :
             base(compilation, syntax, symbol, rootBinder, parentSemanticModelOpt, speculatedPosition)
         {
+            Debug.Assert(!(syntax is ConstructorInitializerSyntax));
```

Comment:
Is the comment above stale? (Does option `(b)` still apply?)

Reply:
> Is the comment above stale? (Does option (b) still apply?)

The option no longer applies, I'll adjust the comment.

---
In reply to: [169830284](https://github.com/dotnet/roslyn/pull/24918#discussion_r169830284) [](ancestors = 169830284)